cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +312,14 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@pytest.mark.meta(automates=[1528859])
```

Comment:
Also needed here
`@test_requirements.rhev`

and nice to have
`@pytest.mark.rhv3`